Dockerfile

```
@@ -1,22 +1,41 @@
 # Base will install runtime dependencies and configure generics
-FROM node:12.16
+FROM node:12-slim as base
+LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>, Nico Stapelbroek <discord-soundbot@nstapelbroek.com>"
+
+RUN apt-get update -q && apt-get install -qy wget
+RUN wget -qO /tini https://github.com/krallin/tini/releases/download/v0.18.0/tini-$(dpkg --print-architecture) && \
+    chmod +x /tini && mkdir /app && chown -R node:node /app
 
-LABEL maintainer="Marko Kajzer <markokajzer91@gmail.com>"
 WORKDIR /app
 
-# Copy some files
-COPY package.json yarn.lock ./
-COPY . /app
 
-# Install ffmpeg and other deps
-RUN apt-get update
-RUN apt-get install --yes build-essential git ffmpeg make python > /dev/null
-RUN yarn install --silent
+# Builder will install system dependencies
+FROM base as builder
+RUN apt-get update -q && apt-get install -qy python make g++ tar xz-utils
```

Turns out `-q` and even `-qq` is not fully quiet, so let's pipe to /dev/null as well.

```suggestion
RUN apt-get update -q > /dev/null
RUN apt-get install --qq --yes python make g++ tar xz-utils > /dev/null
```

RUN apt-get update > /dev/null
RUN apt-get install --yes -qq build-essential ffmpeg make python > /dev/null

---

I think it's okay to have a little bit of output. I wanted to suppress the endless `unpacking` etc. I'll add the --qq as it should suppress most of the output except the warnings.

---

It's still a bit noisy but I think thats okay. As for `/var/lib/apt/lists`: I'm assuming thats [taken care of](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105).